service/awsclusterconfig/v1/version_bundle.go

```diff
@@ -0,0 +1,20 @@
+package v1
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundle() versionbundle.Bundle {
+	return versionbundle.Bundle{
+		Changelogs:   []versionbundle.Changelog{},
+		Components:   []versionbundle.Component{},
```

Comment:
I'm unsure what should be defined here?

Reply:
I think a component this framework provides on a business logic level is the aws-operator. The version restriction is a bit odd though since the aws-operator itself does not have a version, but only version bundles itself. I would say lets mock this with something like `>= 1.0.0`

Reply:
If you meant to literally add component:
```go
{
    Name: "aws-operator",
    Version: ">= 1.0.0",
},
```

This version doesn't seem to be compliant with `versionbundle.Component` rules (I tested with these https://github.com/giantswarm/versionbundle/blob/master/component_test.go), but I'll set it as `1.0.0` for now. I think we can iterate from here when we progress with other parts of `cluster-operator`.

Reply:
Totally. Agree.